SECURITY.md

```diff
@@ -12,3 +12,4 @@ as possible.
 Our email eddress for security report is:
 
 * [xrdp-core@googlegroups.com](mailto:xrdp-core@googlegroups.com)
+
```

Comment:
Is this an intended change?

Reply:
Ah no, but I think I should make it so: see the typo aboven and does this email list actually exist? I can only find xrdp-devel.

Reply:
Yes, the mailing list certainly exists. It is not a public list because the purpose of the list is to receive security (vulnerability) reports privately.

Reply:
Ah gotcha. It might be nice to make that evident as the name of the maling list doesnt make that very clear (=assumption)

Reply:
Will do, thanks for the point.